src/main/java/org/gridsuite/filter/utils/expertfilter/ExpertFilterUtils.java

```diff
@@ -119,11 +126,13 @@ private static String getShuntCompensatorFieldValue(FieldType field, ShuntCompen
                 MAX_SUSCEPTANCE,
                 SWITCHED_ON_SUSCEPTANCE -> getSectionBasedFieldValue(field, shuntCompensator);
             case CONNECTED -> getTerminalFieldValue(field, shuntCompensator.getTerminal());
+            case FREE_PROPERTIES -> shuntCompensator.getProperty(propertyName);
```

Comment:
not needed as we add a case in getFieldValue

Reply:
done